Makefile.PL

```diff
@@ -27,15 +26,15 @@ use File::Spec;
 # Returns: Distribution version as a string
 sub dist_version {
     my ($path) = @_;
-    open(my $fh, '<', $path);
+    open(my $fh, '<', $path)
+        or die "$0: cannot open lib/Pod/Man.pm: $!\n";
     while (defined(my $line = <$fh>)) {
         if ($line =~ m{ \A package \s+ \S+ \s+ (v[\d.]+) }xms) {
-            close($fh);
+            close($fh) or die "$0: cannot close lib/Pod/Man.pm\n";
             return $1;
         }
     }
-    close($fh);
-    die "$0: cannot find version in lib/Pod/Man.pm\n";
```

Comment:
This seems like it is supposed to unconditionally die if this line is reached, so this die shouldn't be removed.

Reply:
I'll let @rra make the call on that.

Reply:
Yeah I agree with Grinnz